src/Discovery/src/ClientCore/SimpleClients/NoOpDiscoveryClient.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Extensions.Logging;
+using Steeltoe.Common;
+using Steeltoe.Common.Discovery;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Steeltoe.Discovery.Client
+{
+    internal class NoOpDiscoveryClient : IDiscoveryClient
+    {
+        internal NoOpDiscoveryClient(ILogger<NoOpDiscoveryClient> logger = null)
+        {
+            logger?.LogWarning("A NoOp Discovery Client has been created! Running in container: {IsContainerized}", Platform.IsContainerized);
```

Comment:
nit ... The "!" seems redundant in that the message is warning level

Reply:
I think in general this message is not clear.  As a user, I would be confused as to why this was a warning.  I think it is good to inform why this is being used.  An example message that would be helpful 
-- Information message: "No discovery client has been configured, using default no-op discovery client." 
-- Information message 2: "Running in container: {IsContainerized}".

Reply:
```suggestion
            logger?.LogWarning("No discovery client has been configured, using default no-op discovery client.");
            logger?.LogInformation("Running in container: {IsContainerized}", Platform.IsContainerized);
```